Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM golang:1.19.1-alpine3.16 AS builder
+
+WORKDIR /app
+RUN set -eux; \
```

Comment:
Do you mind adding a comment in the code explaining why all these deps should be installed?